packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -163,7 +163,7 @@ type RenderProps = {
      * Shapes (points, chords, etc) displayed on the graph that cannot be moved
      * by the user.
      */
-    lockedFigures?: ReadonlyArray<LockedFigure>;
+    lockedFigures: ReadonlyArray<LockedFigure>;
```

Comment:
Love that these parsers are giving us the ability to make the production code safer/cleaner!